File: example/src/main/java/com/example/dumchev/delegateadapters/MainActivity.java
package com.example.dumchev.delegateadapters;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.example.dumchev.delegateadapters.base.KotlinBaseExampleActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        startActivity(new Intent(this, KotlinBaseExampleActivity.class));
    }
}

File: example/src/main/java/com/livermor/dumchev/delegateadapters/MainActivity.kt
package com.livermor.dumchev.delegateadapters

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.livermor.dumchev.delegateadapters.base.BaseExampleActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        startActivity(Intent(this, BaseExampleActivity::class.java))
    }
}